x/epochs/keeper/abci_test.go

```diff
@@ -10,146 +10,98 @@ import (
 	simapp "github.com/osmosis-labs/osmosis/v7/app"
 	"github.com/osmosis-labs/osmosis/v7/x/epochs/types"
 
-	sdk "github.com/cosmos/cosmos-sdk/types"
-)
-
-func TestEpochInfoChangesBeginBlockerAndInitGenesis(t *testing.T) {
-	var app *simapp.OsmosisApp
-	var ctx sdk.Context
-	var epochInfo types.EpochInfo
+	"golang.org/x/exp/maps"
 
-	now := time.Now()
+	"github.com/osmosis-labs/osmosis/v7/osmoutils"
+)
 
-	tests := []struct {
-		expCurrentEpochStartTime   time.Time
-		expCurrentEpochStartHeight int64
-		expCurrentEpoch            int64
-		expInitialEpochStartTime   time.Time
-		fn                         func()
+// This test is responsible for testing how epochs increment based off
+// of their initial conditions, and subsequent block height / times.
+//
+// TODO: Make a new test for init genesis logic
+func (suite KeeperTestSuite) TestEpochInfoBeginBlockChanges() {
+	block1Time := time.Unix(1656907200, 0).UTC()
+	// We run all tests with the same identifier and same duration
+	defaultIdentifier := "hourly"
+	defaultDuration := time.Hour
+	eps := time.Nanosecond
```

Comment:
```suggestion
	const (
		defaultIdentifier = "hourly"
		defaultDuration   = time.Hour
		eps               = time.Nanosecond
	)
```

Reply:
Why?

Reply:
One reason is to avoid the possibility of modifying by accident. Another reason is for semantic alignment with never-changing values. If a value is not const, there is a cognitive overhead to look for places where it changes. So it makes it easier to identify that these never change.

Reply:
But this is a test? Not global variables (not even variables scoped to the entire file)

Reply:
Ah, thats actually fair to make it consts, I'll do it directly!

Reply:
But the variables being defined inside the test do not change the fact that they are meant to be never-changing, right? 

Since there is a language construct to represent the "constness" that is signified here, why would we limit ourselves to only using it in the context of global scope or outsides the tests?